src/imitation/util/logger.py

```diff
@@ -0,0 +1,200 @@
+import contextlib
+import os
+from typing import ContextManager, Optional, Sequence
+
+import stable_baselines.logger as sb_logger
+
+
+def _build_output_formats(folder: str,
+                          format_strs: Sequence[str] = None,
+                          ) -> Sequence[sb_logger.KVWriter]:
+  """Build output formats for initializing a Stable Baselines Logger.
+
+  Args:
+    folder: Path to directory that logs are written to.
+    format_strs: An list of output format strings. For details on available
+      output formats see `stable_baselines.logger.make_output_format`.
+  """
+  os.makedirs(folder, exist_ok=True)
+  output_formats = [sb_logger.make_output_format(f, folder)
+                    for f in format_strs]
+  return output_formats
+
+
+class _AccumulatingLogger(sb_logger.Logger):
```

Comment:
Do we need this class, or can we just use `sb_logger.Logger` directly?

It seems the key difference is that `logkv` also calls `logkv_mean` in `self.mean_logger`. But could we just change `logkv` in `_HierarchicalLogger` to do this directly?

Reply:
yes I think they can be merged (I'm fine either way)

Reply:
(Will revisit this after we make a decision on whether to keep the "raw/sub_dir" and "mean/..." logging key prefixes.)

Reply:
Merged the classes